docs/proposals/15-manage-bastions-and-ssh-key-pair-rotation.md

```diff
@@ -222,3 +222,7 @@ Currently, the `ssh` key pair for the shoot nodes are created once during shoot
         - The "execution" script includes also the original user data script, which it writes to `PATH_CLOUDCONFIG`, compares it against the previous cloud config and runs the script in case it has changed
         - Running the [original user data](https://github.com/gardener/gardener/tree/master/pkg/operation/botanist/component/extensions/operatingsystemconfig/original) script will also run the `gardeneruser` component, where the `authorized_keys` file will be updated
         - After the most recent cloud-config user data was applied, the "execution" script annotates the node with `checksum/cloud-config-data: <cloud-config-checksum>` to indicate the success
+
+### Limitations
+
```

Comment:
```suggestion
### Limitations

Each operating system has its own default user (e.g. `core`, `admin`, `ec2-user` etc). These users get their ssh keys during VM creation (however there is a different handling on Google Cloud Platform as stated below). These keys currently do not get rotated respectively are not removed from the `authorized_keys` file. This means, that the initial `ssh` key will still be valid for the default operating system user.

```

@vpnachev is this correct?

Reply:
More or less yes. 

GCP is really different in how the OS users are manged. There is even no default user, except the `root`. But I like the they it is written now and it should not dig into so much details of how GCP is doing it.